### PR TITLE
Text search

### PR DESCRIPTION
**Pull request by navstyachka**
Demo: [http://navstyachka.com/composer/test/suites/0.html](url)

### CODE REVIEW

File: html-report/src/components/SearchBar.js [hunk 1]
```diff
@@ -1,21 +1,128 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import elasticlunr from 'elasticlunr';
 
+const SEARCH_FIELDS = ['package_name', 'class_name', 'name', 'id', 'status'];
+const SEARCH_REF = 'id';
+const EL_SEARCH = elasticlunr();
+const STATUSES = ['failed', 'ignored'];
```
Comment (artem-zinnatullin) [inline html-report/src/components/SearchBar.js]: lol, this is not what I meant :)

I meant that by clicking on existing `Passed`, `Failed`, `Ignored` blocks search bar should filter tests with clicked status:
screen shot 2017-06-19 at 09 23 25

Comment (navstyachka) [inline html-report/src/components/SearchBar.js]: @artem-zinnatullin I know, but this was the quickest implementation after yesterday's discussion. To code it like you request I need a some more time :)

Comment (artem-zinnatullin) [inline html-report/src/components/SearchBar.js]: Well, maybe remove `Failed | Ignored` for now and add proper implementation later? :)

I mean it's not very obvious and may confuse people and they could even get used to it :D

Comment (navstyachka) [inline html-report/src/components/SearchBar.js]: Yes, captain ⚓️

File: html-report/src/components/Suite.js [hunk 2]
```diff
@@ -32,21 +33,25 @@ export default class Suite extends Component {
...
       <div className="content margin-top-20">
         <div className="title-common"><a href={ paths.fromSuiteToIndex }>Suits list</a>/ Suite {data.id}</div>
         <div className="row justify-between">
-          <div className="card card-info">
+          <div className="card card-info filter-card" onClick={ () => this.filterByStatus('passed') }>
             <div className="text-sub-title-light">Passed</div>
             <div className="card-info__content status-passed">{ data.passed_count }</div>
           </div>
-          <div className="card card-info">
+          <div className="card card-info filter-card" onClick={ () => this.filterByStatus('failed') }>
```
Comment (artem-zinnatullin) [inline html-report/src/components/Suite.js]: yis yis yis

**Pull request merged**